src/app/myAddresses/tests/myAddresses.spec.js

```diff
@@ -1,53 +1,38 @@
 describe('Component: myAddresses', function(){
-    var scope,
-        q,
-        oc,
-        state
-        ;
-    beforeEach(module('orderCloud'));
-    beforeEach(module('orderCloud.sdk'));
-    beforeEach(inject(function($rootScope, $q, OrderCloud, $state){
-        scope = $rootScope.$new();
-        q = $q;
-        oc = OrderCloud;
-        state = $state;
+    var _ocMyAddresses;
+    var address;
+    beforeEach(inject(function(ocMyAddresses){
+        _ocMyAddresses = ocMyAddresses;
```

Comment:
Can you add this service to the global.spec.js file? Be sure to follow the naming conventions established there